### PR TITLE
feat: created shell of function that plots boxplot

### LINKED ISSUE #7: Create boxplot function

### PR DESCRIPTION
**Pull request by miyer26**

### CODE REVIEW

File: src/magmaviz/boxplot.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+    Returns
+    -------
+    altair.vegalite.v4.api.Chart
+        Boxplot displaying distribution of categorical variables with/without faceting
+
+    Examples
+    --------
+    >>> from magmaviz.magmaviz import boxplot
+    >>> plot_words(mtcars, horsepower, cars, facet=True)
```
Comment (iamMoid) [inline src/magmaviz/boxplot.py]: I think you should update this to "color_boxplot" instead of "plot_words"

File: src/magmaviz/boxplot.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    Parameters
+    ----------
+    df : dataframe
+        Dataframe containing the variables for plotting
+    x : string
+        Column-name of the numerical variable to view the distribution of
+    y : list
+        Column names of the categorical variables to assign boxes to
+    facet : boolean
+        Determines whether seprate graphs will be created for each category
```
Comment (iamMoid) [inline src/magmaviz/boxplot.py]: Update to "separate"

File: src/magmaviz/boxplot.py [hunk 3]
```diff
@@ -0,0 +1,24 @@
+def color_boxplot(df, x, y, facet=False):
+    """Plot a boxplot with the magma color scheme and an option to facet.
+
+    Parameters
+    ----------
+    df : dataframe
+        Dataframe containing the variables for plotting
+    x : string
+        Column-name of the numerical variable to view the distribution of
```
Comment (iamMoid) [inline src/magmaviz/boxplot.py]: Remove the "-"

**Review by iamMoid — CHANGES_REQUESTED** [review]
Couple of minor corrections and good to go.

File: src/magmaviz/boxplot.py (hunk 1 above)
Comment (miyer26) [reply src/magmaviz/boxplot.py]: Updated

File: src/magmaviz/boxplot.py (hunk 2 above)
Comment (miyer26) [reply src/magmaviz/boxplot.py]: Updated

File: src/magmaviz/boxplot.py (hunk 3 above)
Comment (miyer26) [reply src/magmaviz/boxplot.py]: Updated

**Review by iamMoid — APPROVED** [review]
looks good!

**Pull request merged**